ospfd/ospf_ase.c

```diff
@@ -748,8 +748,14 @@ void ospf_ase_unregister_external_lsa(struct ospf_lsa *lsa, struct ospf *top)
 
 	if (rn) {
 		lst = rn->info;
-		listnode_delete(lst, lsa);
-		ospf_lsa_unlock(&lsa); /* external_lsas list */
+
```

Comment:
This sure seems like a unrelated bug fix.  Isn't it?

Reply:
This bug was discovered when I implemented this Clear ip ospf process/neighbor feature, hence I have merged it along with this. This is for a crash fix when clear ip ospf process was done multiple times.

Reply:
Please move it into it's own commit with a proper description of what is going on.

Reply:
> Please move it into it's own commit with a proper description of what is going on.

Done.

Reply:
@donaldsharp : The comments have been addressed, pls have a look and approve.